djstripe/models/base.py

```diff
@@ -2,6 +2,7 @@
 import uuid
 from datetime import timedelta
 
+import django
```

Comment:
import django.VERSION would do the trick here, and lighten the import load to boot.

Reply:
from django import VERSION as DJANGO_VERSION, line 5, then
DJANGO_VERSION < (2,1), line 302. 

Apologies for the lazy man's patch, couldn't commit the change against this PR myself.